pkgs/by-name/li/licensed/package.nix

```diff
@@ -0,0 +1,19 @@
+{
+  lib,
+  bundlerApp,
+  bundlerUpdateScript,
+}:
+bundlerApp {
+  pname = "licensed";
+  gemdir = ./.;
+  exes = [ "licensed" ];
+
+  passthru.updateScript = bundlerUpdateScript "licensed";
+
+  meta = {
+    description = "A Ruby gem to cache and verify the licenses of dependencies";
+    homepage = "https://github.com/github/licensed";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.jcaesar ];
```

Comment:
```suggestion
    maintainers = [ lib.maintainers.jcaesar ];
    platforms = lib.platforms.linux;
```

Reply:
I don't particularly mind, but, why? I would assume this works on darwin, but I sure can't test it.

Reply:
I think I saw the ofborg build fail.

Reply:
Oh well. Thank you.